plugin-tester-java/pom.xml

```diff
@@ -13,7 +13,10 @@
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <akka.http.cors.version>0.4.2</akka.http.cors.version>
+    <maven-dependency-plugin.version>3.1.2</maven-dependency-plugin.version>
+    <maven-exec-plugin.version>3.0.0</maven-exec-plugin.version>
+    <akka.http.cors.version>1.1.0</akka.http.cors.version>
+    <akka.grpc.project.version>1.0.2-5-8b94eb72-20201003-1817</akka.grpc.project.version>
```

Comment:
I think we shouldn't encode this in the pom.xml, but pass it in on the command line like `mvn -Dakka.grpc.project.version=1.0.2-5-8b94eb72-20201003-1817`

Reply:
indeed.